Code/Core/FileIO/PathUtils.cpp

```diff
@@ -290,4 +290,39 @@
     outRelativeFileName += pathB;
 }
 
+// GetBaseName
 //------------------------------------------------------------------------------
+
+/* static */ void PathUtils::GetBaseName( const AString& pathName, AString& baseName )
+{
+    const char* slashPos = nullptr;
+    if ( nullptr != ( slashPos = pathName.FindLast( NATIVE_SLASH ) ) )
+        baseName = slashPos + 1;
```

Comment:
minor: FASTBuild's standard never omits braces for if statements.

Reply:
Done.